modules/nw-http2-haproxy.adoc

```diff
@@ -6,17 +6,24 @@
 [id="nw-http2-haproxy_{context}"]
 = Enabling HTTP/2 Ingress connectivity
 
-You can enable transparent end-to-end HTTP/2 connectivity in HAProxy. It allows application owners to make use of HTTP/2 protocol capabilities, including single connection, header compression, binary streams, and more.
+You can enable transparent end-to-end HTTP/2 connectivity in HAProxy. By enabling HTTP/2, an application owner can access HTTP/2 protocol capabilities, which includes single connection, header compression, binary streams, and other features.
 
 You can enable HTTP/2 connectivity for an individual Ingress Controller or for the entire cluster.
 
 To enable the use of HTTP/2 for the connection from the client to HAProxy, a route must specify a custom certificate. A route that uses the default certificate cannot use HTTP/2. This restriction is necessary to avoid problems from connection coalescing, where the client re-uses a connection for different routes that use the same certificate.
 
+[NOTE]
+====
+For a service that runs on a back-end pod, you can set `appProtocol: kubernetes.io/h2c` in the service configuration file, so that the service can send traffic over cleartext routes. You cannot apply this configuration for a service that runs on a front-end pod. 
```

Comment:
Is this considered safe? The Cleartext TCP seems to sending data over a route with no encryption or encoding of any kind.